release/air_tests/air_benchmarks/workloads/tensorflow_benchmark.py

```diff
@@ -330,9 +333,9 @@ def run(
     with open(test_output_json, "wt") as f:
         json.dump(result, f)
 
-    target_ratio = 1.15
+    target_ratio = 1.2
```

Comment:
@krfricke let me know if you have any more insights into this (I ran this 3 times and all were 1.17-1.18x).